### PR TITLE
use current bot parameters for mk4is and 26x24

### PR DESCRIPTION
**Pull request by lmaxwell24**

### CODE REVIEW

**Review by LandonBayer — DISMISSED** [review]

File: networktables.json [hunk 1]
```diff
@@ -6,5 +6,21 @@
     "properties": {
       "persistent": true
     }
+  },
+  {
```
Comment (agelsomini) [inline networktables.json]: Did you intend for these to be part of this PR?  They don't seem drive related

File: commands/drive/robotrelativedrive.py [hunk 2]
```diff
@@ -29,6 +29,6 @@ def execute(self) -> None:
             self.forward(),
             self.sideways(),
             self.rotation()
-            * Preferences.getFloat("Robot Relative Sensitivity", 0.2),  # better control
+            * Preferences.getFloat("Robot Relative Sensitivity", 0.8),  # better control
```
Comment (agelsomini) [inline commands/drive/robotrelativedrive.py:32]: Out of curiosity, is this just something you adjusted by feel on the actual bot?

File: networktables.json (hunk 1 above)
Comment (lmaxwell24) [reply networktables.json]: haha no, stuff I didn't push to armsubsystem

File: commands/drive/robotrelativedrive.py (hunk 2 above)
Comment (lmaxwell24) [reply commands/drive/robotrelativedrive.py:32]: yeah, we don't use robot relative very often (if ever) so having higher defaults makes more sense for a systems test, the preferences allow it to be tuned down

**Review by agelsomini — DISMISSED** [review]

**Review by LandonBayer — APPROVED** [review]

**Review by agelsomini — APPROVED** [review]

**Pull request merged**